### PR TITLE
Client maktub develop ruan

### PR DESCRIPTION
**Pull request by Flame1994**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -16,7 +16,17 @@
...
 			</section-image>
-			<div style="height: 400px;">
+			<section-image-text title="Provided Title" subtitle="Provided subtitle" image="https://i.imgur.com/bX2Uk7C.jpg" image-location="right"> </section-image-text>
+			<div style="position:relative;">
+				<div style="position: absolute; bottom: 0; left: 0; width: 100%;">
+					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 500">
+						<path
+							fill="#FB9062"
+							fill-opacity="1"
+							d="M0,128L80,133.3C160,139,320,149,480,133.3C640,117,800,75,960,80C1120,85,1280,139,1360,165.3L1440,192L1440,500L1360,500C1280,500,1120,500,960,500C800,500,640,500,480,500C500,500,160,500,80,500L0,500Z"
```
Comment (angelomelonas) [inline src/App.vue:26]: Is this the vector wave thingy at the bottom?

**Review by angelomelonas — APPROVED** [review]
NICE

**Comment by Flame1994** [comment]
Just to note, we can fix the inline styling in App.vue a bit later.

File: src/App.vue (hunk 1 above)
Comment (Flame1994) [reply src/App.vue:26]: Yeeees

**Pull request merged**